Sources/XcodeProj/Objects/Project/PBXProj.swift

```diff
@@ -52,6 +52,24 @@ public final class PBXProj: Decodable {
         }
     }
 
+    /// Initializes the project with a path to the pbxproj file.
+    ///
+    /// - Parameters:
+    ///   - path: Path to a pbxproj file.
+    public convenience init(path: Path) throws {
+        let pbxproject: PBXProj = try PBXProj.createPBXProj(path: path)
```

Comment:
nit: worth naming this as `pbxproj` to avoid confusion - there is another [`PBXProject`](https://github.com/tuist/XcodeProj/blob/main/Sources/XcodeProj/Objects/Project/PBXProject.swift) type in the code base which is slightly different 🤐